recipes/methylextract/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "1.9.1" %}
+{% set sha256 = "f9c1cb9d01814745cd37384790ee6bf71dfa69030414a11eda728fcdcc505d40" %}
+
+package:
+  name: methylextract
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: http://bioinfo2.ugr.es/MethylExtract/downloads/MethylExtract.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  run:
+    - perl
+    - samtools
+
+about:
+  home: http://bioinfo2.ugr.es/MethylExtract/
+  license: GNU General Public License v3 (GPLv3)
+  license_family: GPL3
+  summary: High-Quality methylation maps and SNV calling from whole genome bisulfite sequencing data
+
+test:
+  commands:
+    - which MethylExtract.pl
```

Comment:
Can you run the scripts as well? With some `--version` or `--help` flag?

Reply:
Thanks for your review. 
Unfortunately, the scripts don't have such flags. And running them without arguments results in exitcode 255, which makes the test fail.

I guess another approach would be to grep for some string in the usage information that gets printed when calling the script, if you prefer that over 'which'.